src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb

```diff
@@ -399,6 +400,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 If IsScriptRunner Then
                     Select Case name
+                        Case "i", "i+"
+                            If value IsNot Nothing Then Exit Select
```

Comment:
If that is the only reason, I think we should be consistent with the surrounding code.